cartography/data/jobs/cleanup/gcp_bigquery_access_cleanup.json

```diff
@@ -0,0 +1,29 @@
+{
+	"statements": [
+		{
+			"query": "MATCH (n:GCPBigqueryTable)<-[:HAS_TABLE]-(:GCPBigqueryDataset)<-[:RESOURCE]-(:GCPProject{id: $GCP_PROJECT_ID})<-[:OWNER]-(:CloudanixWorkspace{id: $WORKSPACE_ID}) WHERE n.lastupdated <> $UPDATE_TAG WITH n LIMIT $LIMIT_SIZE DETACH DELETE (n) return COUNT(*) as TotalCompleted",
```

Comment:
add queries for `GCPAccess`

Reply:
and implement this in the file `gcp_bigquery_cleanup.json`

Reply:
please check i have made the changes